extension/content.js

```diff
@@ -344,6 +344,21 @@ function addFilterCommentsByYou() {
 	}
 }
 
+function addProjectNewLink() {
+	const projectNewLinkId = 'refined-github-project-new-link';
+	const projectNewLink = `<a href="/${repoUrl}/projects/new" class="btn btn-sm" id="${projectNewLinkId}">Add a project</a>`;
+	if ($(`#${projectNewLinkId}`).length === 0) {
```

Comment:
We'd need to add the button only if the Projects checkbox is checked. Otherwise, the link is pointing to a 404 when the Projects are not enabled. I guess we can skip adding it when you are just checking the checkbox. I don't think it's worth it.